app/grants/urls.py

```diff
@@ -27,9 +27,9 @@
     get_interrupted_contributions, get_replaced_tx, grant_activity, grant_categories, grant_details, grant_details_api,
     grant_details_contributions, grant_details_contributors, grant_edit, grant_fund, grant_new, grant_new_whitelabel,
     grants, grants_addr_as_json, grants_bulk_add, grants_by_grant_type, grants_cart_view, grants_info,
-    grants_stats_view, invoice, leaderboard, manage_ethereum_cart_data, new_matching_partner, profile, quickstart,
-    remove_grant_from_collection, save_collection, subscription_cancel, toggle_grant_favorite, verify_grant,
-    ingest_contributions_view, ingest_contributions
+    grants_stats_view, ingest_contributions, ingest_contributions_view, invoice, leaderboard, manage_ethereum_cart_data,
+    new_matching_partner, profile, quickstart, remove_grant_from_collection, save_collection, subscription_cancel,
+    toggle_grant_favorite, verify_grant,
```

Comment:
Where are these imports used?

Reply:
that's auto linting by pre-commit hook on ```grants/urls.py```. they're used there.

not related to this PR